### ISSUE TITLE
Confusion about Boston Local vs OV IDs and thumbs missing thumbnail

### ISSUE THREAD

**Issue by mccalluc** [WGBH/openvault3#378]
thumbnails were dumped into s3 catalog for which there is no corresponding OV record
- https://s3.amazonaws.com/openvault.wgbh.org/catalog/asset_thumbnails/V_UDAMVZGA4JEY06N.png exists
- but http://ov-qa1.wgbh-mla-test.org/catalog/V_UDAMVZGA4JEY06N is 404

(Note that OV IDs are longer, and consist only of hex digits.)

This will be confusing over the long run.
- The only thumbs under s3 catalog should be jpgs with OV IDs as their name
- If that is not the case, they should go somewhere else.

S3 should be cleaned up, and there should be another PR pointing to the correct locations, and all 6 images should 4. (Only 5 do right now.)

**Comment by caseyedavis12** [comment]
@mccalluc: Fixed. Went back to my original plan and put the images in the special collections folder.

**Comment by mccalluc** [comment]
Waiting on PR.

**Comment by caseyedavis12** [comment]
@mccalluc: it's the fix-urls-1 and fix-urls PRs. (one for Open Vault and other for AAP[&#X1f60e]

**Comment by mccalluc** [comment]
looks good.
